### ISSUE TITLE
TypeScript build failing due to external package update

### ISSUE THREAD

**Issue by chrisvxd** [transitive-bullshit/create-react-library#73]
Hi @transitive-bullshit 👋 

## What I did

Generated a TypeScript project.

## What I expected

The generated project to build correctly with rollup after installing.

## What happened

I received the error when building with rollup

```
[!] Error: 'rollup-plugin-postcss-modules' provides a 'postcss-modules' plugin and its `getJSON()`. You cannot specify `modules.getJSON`
```

I tracked it down to `rollup-plugin-postcss-modules` which introduced a breaking change in `v1.0.9`, released a few hours ago. I'm tracking that [here](https://github.com/flying-sheep/rollup-plugin-postcss-modules/issues/6).

Also, I notice you're not using this package in the default template. Is this deliberate, or an oversight?

**Comment by transitive-bullshit** [comment]
> Also, I notice you're not using this package in the default template. Is this deliberate, or an oversight?

The default / official [rollup-plugin-postcss](https://github.com/egoist/rollup-plugin-postcss) handles things correctly; If I'm understanding the TypeScript side of things, there are some minor issues between rollup and TS that `rollup-plugin-postcss-modules` is aimed at resolving. This package definitely seems less standardized and well maintained than `rollup-plugin-postcss`.

To be honest, the TypeScript template hasn't received a lot of love, so I wasn't even aware that we were using such a haphazard dependency on that side of things.

**Comment by transitive-bullshit** [comment]
Just published your fix in `v2.6.5`

**Comment by transitive-bullshit** [comment]
Closing for now; feel free to reopen if you're still seeing issues 😄
